src/UnityExplorer.sln

```diff
@@ -48,6 +50,32 @@ Global
 		{B21DBDE3-5D6F-4726-93AB-CC3CC68BAE7D}.Release_STANDALONE_Mono|Any CPU.ActiveCfg = STANDALONE_Mono|Any CPU
 		{B21DBDE3-5D6F-4726-93AB-CC3CC68BAE7D}.Release_STANDALONE_Mono|Any CPU.Build.0 = STANDALONE_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE_CoreCLR|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE_CoreCLR|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE_Cpp|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE_Cpp|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE_Unity_Cpp|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE_Unity_Cpp|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE5_Mono|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE5_Mono|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE6_Mono|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE6_Mono|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE6_Unity_Mono|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_BIE6_Unity_Mono|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Cpp_CoreCLR|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Cpp_CoreCLR|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Cpp_net472|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Cpp_net472|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Cpp_net6preview|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Cpp_net6preview|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Mono|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_ML_Mono|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_STANDALONE_Cpp_CoreCLR|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_STANDALONE_Cpp_CoreCLR|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_STANDALONE_Cpp|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_STANDALONE_Cpp|Any CPU.Build.0 = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_STANDALONE_Mono|Any CPU.ActiveCfg = Release_Mono|Any CPU
+		{E3A1056F-8C8C-4394-A836-4803D6208AEA}.Release_STANDALONE_Mono|Any CPU.Build.0 = Release_Mono|Any CPU
```

Comment:
Is this change necessary?

Reply:
This was done automatically by my IDE, so I don't really know.

Reply:
Okay, please revert before the merge for consistency in build logic, etc.